libuavcan/CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ endif ()
 #
 execute_process(COMMAND ${PYTHON} setup.py build WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/dsdl_compiler OUTPUT_QUIET)
 set(DSDLC_INPUTS "test/dsdl_test/root_ns_a" "test/dsdl_test/root_ns_b" "${CMAKE_CURRENT_SOURCE_DIR}/../dsdl/uavcan")
-set(DSDLC_OUTPUT "include/dsdlc_generated")
+set(DSDLC_OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/include/dsdlc_generated")
```

Comment:
There seems to be a mixture of `${CMAKE_CURRENT_BINARY_DIR}` and `${CMAKE_BINARY_DIR}` being used here -- will this cause any issues?